recipes/client.rb

```diff
@@ -18,14 +18,19 @@
 # limitations under the License.
 #
 
-nt_version = ::Windows::VersionHelper.nt_version(node)
-
-if nt_version == 6.1
+def netfx_install
   windows_feature 'NetFx3' do
     action :install
   end
 end
 
+case node['platform_version']
+when '6.1.7600'
```

Comment:
You can simplify this one a bit with node['platform_version'].to_f if you're goal is just to catch everything on 6.1